CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## securesystemslib v0.16.0
+
+### Added
+* Added new, self explanatory, AnyNonEmptyString schema (#244)
```

Comment:
Not sure if the hyphen is mandatory, or if this is an AE/BE thing...

```suggestion
* Added new, self-explanatory, AnyNonEmptyString schema (#244)
```

Reply:
You are correct!